tsconfig.json

```diff
@@ -24,13 +24,6 @@
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true
   },
-  "include": ["src/**/*", "types/**/*"],
-  "exclude": [
-    ".github",
-    ".vscode",
-    "dist",
-    "node_modules",
-    "test",
-    "./*.js"
-  ]
+  "include": ["src/**/*", "types/**/*", "*.js"],
+  "exclude": [".github", ".vscode", "dist", "node_modules", "test"]
```

Comment:
if you exclude all of node_modules would that prevent any of the typings from the infra repo from working?

Reply:
No, this has worked fine before, I believe this just tells the TypeScript not to scan them to see if they align with our rules.